### ISSUE TITLE
Loss Function to Penalize Low Bias

### ISSUE THREAD

**Issue by jordancaraballo** [nasa-nccs-hpda/above-shrubs.pytorch#1]
Loss Function to Penalize Low Bias
